components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -249,6 +254,13 @@ function OrgSettingsForm(props: { org?: Organization; isOwner: boolean }) {
                     onClose={() => setShowImageEditModal(false)}
                 />
             )}
+
+            <Heading2 className="pt-12">Maximum Workspace Size</Heading2>
+            <Subheading className="max-w-2xl">
+                Limit the maximum workspace size available in your organization.
+            </Subheading>
+
+            <WorkspaceSizeOptions settings={settings} />
```

Comment:
I think, this action should be accessible to owners only? We should disable this if a Member is coming to Organization settings (just like Workspace sharing settings)

```suggestion
            <WorkspaceSizeOptions settings={settings} disabled={!isOwner} />
```